donate.html

```diff
@@ -240,9 +240,9 @@ <h2 class="donate-option3-header">$5</h2>
 
   <!-- secondary links -->
   <section class="secondary-links">
-      <a class="nav-link" target="_blank" href="./files/brandingGuide.pdf">Branding</a>
-      <p>•</p>
-      <a class="nav-link" target="_blank" href="./files/terms.pdf">Code of Conduct</a>
+      <a class="nav-link" target="_blank" href="./files/brandingGuide.pdf" style="color: #000 !important;">Branding</a>
+      <p style="color: #000 !important;">•</p>
+      <a class="nav-link" target="_blank" href="./files/terms.pdf" style="color: #000 !important;">Code of Conduct</a>
   </section>
 
   <!-- footer -->
```

Comment:
in other pages, the footer was inside a container which is why I had to hardcode a 15px offset (you can see this in main.css). since that's not the case here, can you add something like "margin-left: 0px !important;" for the footer on this page?